### PR TITLE
Validate that n of m elections are encrypting and decrypting correctly

### PR DESCRIPTION
**Pull request by lprichar**
### Issue

Address [#30 ](https://github.com/microsoft/electionguard-program/issues/30)

### Description
Simply adds a unit test to validate that n of m contests are correctly encrypting and decrypting all votes.

### Testing
Run the tests

### CODE REVIEW

File: bindings/netstandard/ElectionGuard/ElectionGuard.Encryption.Tests/TestEncrypt.cs [hunk 1]
```diff
@@ -2,7 +2,7 @@
 using ElectionGuard.Encryption.Utils;
 using NUnit.Framework;
 
-namespace ElectionGuard.Encrypt.Tests
+namespace ElectionGuard.Encryption.Tests
```
Comment (lprichar) [inline bindings/netstandard/ElectionGuard/ElectionGuard.Encryption.Tests/TestEncrypt.cs:5]: I would eventually like to change all namespaces, but for now just boy scouting the files I change

File: bindings/netstandard/ElectionGuard/ElectionGuard.Encryption.Tests/TestEncrypt.cs [hunk 2]
```diff
@@ -189,13 +222,27 @@ public void Test_EncryptMediator_Hashes_Dont_Match()
...
-            catch (Exception ex)
+            catch (Exception)
             {
                 // if there is an exception then the manifest hash would not be equal
                 Assert.AreNotEqual(context.ManifestHash.ToHex(), internalManifest.ManifestHash.ToHex());
             }
         }
 
 
+        private static CiphertextBallotContest FindContestOrDefault(CiphertextBallot ballot, Func<CiphertextBallotContest, bool> func)
```
Comment (lprichar) [inline bindings/netstandard/ElectionGuard/ElectionGuard.Encryption.Tests/TestEncrypt.cs:234]: I was tempted to put this into the Encryption project, but wasn't sure if we're trying to keep that clean

Comment (SteveMaier-IRT) [reply bindings/netstandard/ElectionGuard/ElectionGuard.Encryption.Tests/TestEncrypt.cs:234]: This would be a great enhancement for v2.

**Review by SteveMaier-IRT — APPROVED** [review]
looks good.

**Pull request merged**